src/Bugsnag/Payload/Report.cs

```diff
@@ -7,7 +7,7 @@ namespace Bugsnag.Payload
 {
   public class Report : Dictionary<string, object>, IPayload
   {
-    private static readonly string _payloadVersion = "4";
+    private static readonly string _payloadVersion = "4.0";
```

Comment:
I _think_ either way of specifying this works.

Reply:
I got this from maze-runner itself, so I'll update maze-runner

Reply:
Added additional checks in https://github.com/bugsnag/bugsnag-dotnet/pull/94/commits/ed3e8c2b1a2b837e0d5c0e5655c12b2beaa84b38, these could/should probably be added to the `maze-runner` repo itself as they are quite platform neutral

Reply:
I'll remove this once the `maze-runner` PR is merged

Reply:
Looks good. If we add it to the mazerunner repo itself we would need to make the severityReason generic.